satpy/demo/_google_cloud_platform.py

```diff
@@ -24,7 +24,7 @@
     from urllib.request import urlopen
 except ImportError:
     # python 2
-    from urllib2 import URLError, urlopen
+    from urllib2 import URLError, urlopen  # type: ignore
```

Comment:
We are not supporting python2 anymore, right?

Reply:
Ah yes, good catch. I missed that. I'll update that.